Answers.md

```diff
@@ -2,10 +2,17 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+    It is a library of JS and it handles large amounts of data and potentially from multiple users and displays to the DOM without bogging down the DOM.

```

Comment:
There is more to it. The most important concept is that React has "Virtual DOM" which only updates things that changed, because of that it is more efficient and faster.

Reply:
Hello, this pull request came from a deleted repository and this is not the project I submitted for CS PT3 on Dec 8th. That would be pull request #755 and it is a little better than this one, though all the feedback you gave me on components and thinking in the React way still applies. Thank you.